2-WebApp-graph-user/2-6-BFF-Proxy/CallGraphBFF/appsettings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "AzureAd": {
+    "Instance": "https://login.microsoftonline.com/",
+    "TenantId": "[Enter 'common', or 'organizations' or the Tenant Id (obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",
+    "ClientId": "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
+    "ClientSecret": "[Copy the client secret added to the app from the Azure portal]",
```

Comment:
We could use the ClientCredentials collection insted of ClientSecret and ClientCertificates
(docs still being written though)

Reply:
done! (sorry for taking so long to get back to this)